source/Octo/Commands/NuGetPackageBuilder.cs

```diff
@@ -18,13 +18,19 @@ public NuGetPackageBuilder(IOctopusFileSystem fileSystem, Serilog.ILogger log)
             this.log = log;
         }
 
-        public void BuildPackage(string basePath, IList<string> includes, ManifestMetadata metadata, string outFolder, bool overwrite)
+        public void BuildPackage(string basePath, IList<string> includes, ManifestMetadata metadata, string outFolder, bool overwrite, bool verboseInfo)
         {
             var nugetPkgBuilder = new PackageBuilder();
 
-            nugetPkgBuilder.PopulateFiles(basePath, includes.Select(i => new ManifestFile { Source = i }));
+            var manifestFiles = includes.Select(i => new ManifestFile {Source = i}).ToList();
+            nugetPkgBuilder.PopulateFiles(basePath, manifestFiles);
             nugetPkgBuilder.Populate(metadata);
 
+            if (verboseInfo)
+            {
+                nugetPkgBuilder.Files.ToList().ForEach(f => log.Information("Adding file: {Path}", f.Path));
```

Comment:
Sorry, pet hate of mine is making a list just so you get access to ForEach on it. Just use a `foreach` loop, or if you really want `ForEach` make an extension method on IEnumerable or something.